apps/prairielearn/src/pages/instructorAssessmentStatistics/instructorAssessmentStatistics.ts

```diff
@@ -31,33 +37,52 @@ router.get(
     await updateAssessmentStatistics(res.locals.assessment.id);
 
     // re-fetch assessment to get updated statistics
-    const assessmentResult = await sqldb.queryOneRowAsync(sql.select_assessment, {
-      assessment_id: res.locals.assessment.id,
-    });
-    res.locals.assessment = assessmentResult.rows[0].assessment;
+    res.locals.assessment = await sqldb.queryRow(
```

Comment:
Is there a need to update `res.locals` here? Can this be a local variable passed as argument to the html function? This may affect the partials, but I don't think any information that has been updated between the middleware and this point is needed in any of the partials used in this code.

Reply:
Updated this to a local variable here and passing to the HTML in [fe614f8](https://github.com/PrairieLearn/PrairieLearn/pull/9809/commits/fe614f896046789604dac2796569dc2b7421cebc).